### PR TITLE
chore: update hokusai v0.5.13

### PR DESCRIPTION
**Pull request by izakp**
Update hokusai to version 0.5.13

cc @starsirius @iskounen

### CODE REVIEW

File: src/remote-docker/remote-docker.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Orb Version 0.1.9
+# Orb Version 0.1.10
```
Comment (izakp) [inline src/remote-docker/remote-docker.yml:1]: Wonder if there is a bug in the publish-orb script...
```
artsy/remote-docker@0.1.9 is the latest, cannot publish older version 0.1.1
Please update ./src/remote-docker/remote-docker.yml to have a version greater than 0.1.9
```

Comment (zephraph) [reply src/remote-docker/remote-docker.yml:1]: Hmmm

Comment (zephraph) [reply src/remote-docker/remote-docker.yml:1]: Oh, yep! Yikes.

File: scripts/orb_utils.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-VERSION_REGEX="[0-9]\.[0-9]\.[0-9]"
+VERSION_REGEX="[0-9]*\.[0-9]*\.[0-9]*"
```
Comment (izakp) [inline scripts/orb_utils.sh:3]: @zephraph this should fix the mismatched version
```
$ YML_PATH=$(get_orb_path remote-docker)
$ VERSION_COMMENT=$(head -n 1 "$YML_PATH")
$ echo "$VERSION_COMMENT" | grep -o "$VERSION_REGEX"
0.1.1
$ VERSION_REGEX="[0-9]*\.[0-9]*\.[0-9]*"
$ echo "$VERSION_COMMENT" | grep -o "$VERSION_REGEX"
0.1.10
```

Comment (zephraph) [reply scripts/orb_utils.sh:3]: Thanks for that!

**Comment by zephraph** [comment]
Let's ship it! 🚢. 

Thanks @izakp for improving the version matching.

**Pull request merged**